doc/adrs/187_add_visibility_tool_on_diagram_nodes.adoc

```diff
@@ -0,0 +1,87 @@
+= ADR-187 - Add visibility tool on diagram nodes
```

Comment:
- Nothing in the context of this ADR is specific to the management of the visibility. It's just about adding actions on nodes that can execute model operations and optionally be overridden on the frontend by another React component.

- There's a sizable part missing (which I believe should be in a second ADR) which is the manage visibility dialog which display in a list a subset of the node of the diagram that can be hidden / revealed. This is highly specific to the visibility and will be based on what is defined here (once the extension point of the frontend is available to override the rendering).
  - It needs to display in real time what is in the node and allow to show/hide nodes with a click.
  - It also provides some search mecanism etc
  - This dialog will not be cheap

Reply:
The file has been renamed.

A second ADR will come soon in a separate PR.